cmd/execution_builder.go

```diff
@@ -503,11 +503,11 @@ func (e *ExecutionNodeBuilder) LoadComponentsAndModules() {
 
 			vm := fvm.NewVirtualMachine(rt)
 
-			fvmOptions := append([]fvm.Option{}, node.FvmOptions...)
+			fvmOptions := append([]fvm.Option{fvm.WithLogger(node.Logger)}, node.FvmOptions...)
```

Comment:
Wouldn't it be better to keep a logger as required parameter rather than an option?

Reply:
It is't not a required part for FVM to function, so I think having it as optional might be ok. It was mostly used as noop anyway.

Reply:
Yes, but that makes it easy to miss. FVM seems crucial enough that we should always have a logger, even if its noop.

Reply:
This would cause an inconsistency with
- `NewContext(logger)`
- `context.Child().WithLogger(logger)`

I wanted to avoid that, so its always `.WithLogger(logger)`